crates/uv-fs/src/lib.rs

```diff
@@ -187,10 +187,12 @@ pub fn copy_atomic_sync(from: impl AsRef<Path>, to: impl AsRef<Path>) -> std::io
 }
 
 #[cfg(windows)]
-fn backoff_file_move() -> backoff::ExponentialBackoff {
-    backoff::ExponentialBackoffBuilder::default()
-        .with_initial_interval(std::time::Duration::from_millis(10))
-        .with_max_elapsed_time(Some(std::time::Duration::from_secs(10)))
+fn backoff_file_move() -> backon::ExponentialBackoff {
+    use backon::BackoffBuilder;
+    // 10*2^9 milliseconds = 5 seconds = max 10 seconds of trying
```

Comment:
I feel so dumb, can you explain this math? (I assume the intention is to be unchanged.)

Reply:
we start at 10 milliseconds and try 9 times. the longest try is therefore `10*pow(2, 9)` milliseconds, which is 5 seconds. This gives us about 10 seconds of attempting to retry, because the sum of powers of 2 is just the next power of 2 (-1).

Or more intuitively, we wait about 5 seconds in the worst case, and that dwarfs everything else combined.

Reply:
Updated the comment, thanks for calling it out!